test/remote-httputility.cpp

```diff
@@ -0,0 +1,77 @@
+/* Icinga 2 | (c) 2025 Icinga GmbH | GPLv2+ */
+
+#include <BoostTestTargetConfig.h>
+#include "remote/httputility.hpp"
+#include "test/icingaapplication-fixture.hpp"
+
+using namespace icinga;
+
+BOOST_AUTO_TEST_SUITE(remote_httputility)
+
+BOOST_FIXTURE_TEST_CASE(IsValidHeaderName, IcingaApplicationFixture)
+{
+	// Use string_view literals (""sv) to that tests including \0 are possible.
```

Comment:
Something is missing in this sentence. Maybe you wanted to write `... (""sv) so that tests ...`.